UPGRADE.md

```diff
@@ -2,6 +2,48 @@
 
 ## dev-develop
 
+### Definition of security contexts
+
+The definition of security contexts in the `Admin` classes have changed. They
+used to look like the following example:
+
+```php
+public function getSecurityContexts()
+{
+    return [
+        'Sulu' => [
+            'Media' => [
+                'sulu.media.collections',
+            ],
+        ],
+    ];
+}
+```
+
+Now the available permission types have also to be passed:
```

Comment:
Now you should also pass the permission types that you want to enable in the context: